spring-cloud-deployer-cloudfoundry/pom.xml

```diff
@@ -44,7 +34,6 @@
 		<dependency>
 			<groupId>org.cloudfoundry</groupId>
 			<artifactId>cloudfoundry-client-reactor</artifactId>
-			<version>${cloudfoundry-java-lib.version}</version>
 			<!-- @TODO boot3 remove when updated -->
```

Comment:
Question: Is this TODO still valid?

Reply:
Yes I believe it is. The version of the client still has older annotations lib in it. Will follow up on the CF libs in a subsequent PR.